scripts/gen_vpc_ip_limits.go

```diff
@@ -208,6 +208,24 @@ func addManualLimits(limitMap map[string]awsutils.InstanceTypeLimits) map[string
 		"c5ad.metal":    {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: true},
 		"p4de.24xlarge": {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: false},
 		"c7g.metal":     {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: true},
+		"m7g.medium":    {ENILimit: 2, IPv4Limit: 4, HypervisorType: "unknown", IsBareMetal: false},
```

Comment:
Why are these manually added? Does the script not work for these instance types?

Reply:
yes, these instance types are not available from DescribeInstanceTypes API, so I manually add them

Reply:
Any idea why they are not available or if they will be available in the future? We try to avoid these manual cases unless there is a really good reason

Reply:
They should be available soon(within this week, confirmed by EC2 PM), will update this PR once they are ready.

Reply:
update this file as new instance types are now available.